test/functional/wallet_conflicts.py

```diff
@@ -37,7 +37,6 @@ def run_test(self):
         """
 
         self.test_block_conflicts()
-        self.generatetoaddress(self.nodes[0], COINBASE_MATURITY + 7, self.nodes[2].getnewaddress())
```

Comment:
Linter fails because the import of `COINBASE_MATURITY` is not needed anymore (line 12).

Reply:
Done, thanks